ibis-server/app/model/data_source.py

```diff
@@ -30,6 +32,12 @@
 )
 
 
+class SSLMode(str, Enum):
+    DISABLE = "Disable"
+    REQUIRE = "Require"
+    VERIFY_CA = "Verify CA"
```

Comment:
```suggestion
    DISABLED = "disabled"
    ENABLED = "enabled"
    VERIFY_CA = "verify_ca"
```
I prefer to rename `require` -> `enabled`.
We can use the snake case for the enum value. The request body could be
```
{
      "sslMode": "verify_ca",
      "sslCA": "xxx",
}
```

Reply:
Hi @goldmedal, I have tried adding a test for `ssl_mode=enabled` as shown below:
```python
async def test_connection_ssl_enabled(client, mysql: MySqlContainer):
    connection_info = _to_connection_info(mysql)
    connection_info["ssl_mode"] = "enabled"
    response = await client.post(
        url=f"{base_url}/metadata/version",
        json={"connectionInfo": connection_info},
    )
    assert response.status_code == 200
    assert response.text == '"8.0.40"'
```

However, the test passes, and I'm unsure if this is expected behavior due to MySQL server's default SSL configuration [ref: [link](https://dzone.com/articles/configuring-and-managing-ssl-on-your-mysql-server)].

> By default, MySQL server always installs and enables SSL configuration. However, it is not enforced that clients connect using SSL. Clients can choose to connect with or without SSL as the server allows both types of connections.

Your help is very much appreciated!